client/src/core/ErrorBoundary.js

```diff
@@ -78,24 +83,47 @@ export class ErrorBoundary extends React.Component {
             alignItems: "center",
           }}
         >
+          {redirected_msg ? (
```

Comment:
This is kind of an artifact from when it was redirecting at error boundary level, but it's still applicable (sort of). Because `SafeJSURL.parse(options);` itself throws an error and hits the error boundary for a second before the reload happens.

Just left it in there to see the reaction since it was already written. I can remove it!

Reply:
Do you mean an error in the `ErrorBoundary` itself, or in the redirect code? The `ErrorBoundary` itself is tested by the route load tests. We could add specific testcafe tests for the redirect code too, would be pretty easy.

Reply:
Well, since I just removed SafeJSURL parse, I removed this as well. I guess when it gets stuck reloading, we'll put this back in haha